docs/howto/add_pagination_links.md

```diff
@@ -67,34 +54,50 @@ ex:
 }
 ```
 
-ActiveModelSerializers pagination relies on a paginated collection with the methods `current_page`, `total_pages`, and `size`, such as are supported by both [Kaminari](https://github.com/amatsuda/kaminari) or [WillPaginate](https://github.com/mislav/will_paginate).
-
+ActiveModelSerializers pagination relies on paginated collections which define the methods `#current_page`, `#total_pages`, and `#size`.
+Such methods are supported by both [Kaminari](https://github.com/amatsuda/kaminari) or [WillPaginate](https://github.com/mislav/will_paginate),
+but you can also roll out your own paginated collection by defining these methods.
```

Comment:
Oh, sorry, I meant the entirely of the diff in the docs, not just the comment about rolling your  own :(